python/gzdice/Portfile

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                gzdice
+version             0.1.2
+revision            0
+
+categories          python
+platforms           darwin
+license             MIT
+maintainers         @gleeblezoid \
+                    gleeblezoid.com:gzdice
+
```

Comment:
Since this is Python, it may make sense to specify that this port is not CPU-architecture -specific by adding:
```
supported_archs     noarch
```

The typical values for `supported_archs` would be options like `x86_64` or `arm64`, but since Python code is interpreted, `noarch` specifies that this port does not have a specifically required CPU architecture.